requirements.txt

```diff
@@ -0,0 +1,8 @@
+flask>=0.10.0
+lxml>=3.4.4
+spyne>=2.12.8
+python-dateutil
+jinja2
+pytz
+sqlalchemy
+odm2api
```

Comment:
`suds` seems to be used only for testing so I assumed it is a dev dependency. I can move it here if the main code needs it but I could not find any reference to it.